### PR TITLE
[SVN] Octave multiline input

### PR DESCRIPTION
**Pull request by woutersj**
This PR implements multiline input to Octave sessions and folds.

### CODE REVIEW

**Comment by da-liii** [comment]
Verified:
+ GNU/Linux
+ GNU TeXmacs v1.99.18
+ GNU Octave 4.4.1

File: octave/tm/tmrepl.m [hunk 1]
```diff
@@ -29,6 +33,7 @@ function tmrepl()
       # Reset ans to empty string
       ans= "";
       # Suppress the output
+      __r = strrep(__r,"\n",";\n");
```
Comment (da-liii) [inline octave/tm/tmrepl.m]: This line changes the syntax of the octave script, and it is not necessary.

Comment (da-liii) [reply octave/tm/tmrepl.m]: The global ans will be displayed after the block of code executed. For one line or multi lines, only one ans will be displayed.

**Comment by da-liii** [comment]
Tested:
+ Windows 10
+ GNU TeXmacs: 1.99.18
+ GNU Octave: 5.2.0

**Review by da-liii — CHANGES_REQUESTED** [review]
Works fine, but one line is not needed. I've commented on it.

File: octave/tm/tmrepl.m (hunk 1 above)
Comment (woutersj) [reply octave/tm/tmrepl.m]: Agreed, I have undone this change.

**Review by da-liii — APPROVED** [review]
LGTM now

**Pull request closed without merging**